src/foam/u2/crunch/WizardRunner.js

```diff
@@ -58,6 +58,9 @@ foam.CLASS({
         return (options.inline ?? true) && !! parentWizard;
       }
     },
+    {
+      name: 'controller'
```

Comment:
Would be nice to add some documentation to this

Reply:
ya good call - updated. feel free to make suggestions